src/tools/reverseEvent/reverse-event.ts

```diff
@@ -0,0 +1,73 @@
+import fs from "fs-extra";
+
+type logType = {
+  "_id": string,
+  "rid": string,
+  "command": number,
+  "args": string[]
+}
+
+async function reverseEvent() {
+  try {
+    const logs = await fs.readJson(__dirname + "/eventlog.json");
+
+    let args: {[key: string]: string}[] = [];
+
+    logs.forEach((log: logType) => {
+      let arg: {[key: string]: string} = {
+        "rid": "empty",
```

Comment:
Maybe misunderstanding, what I think is not "empty, it should be "".

Reply:
I'll modify it.